data/test/index.js

```diff
@@ -59,3 +59,26 @@ describe( 'query', () => {
 		expect( tree ).toMatchSnapshot();
 	} );
 } );
+
+describe( 'subscribe', () => {
+	it( 'registers multiple selectors to the public API', () => {
+		let data = null;
+		const store = registerReducer( 'myAwesomeReducer', ( state = 0 ) => state + 1 );
+		registerSelectors( 'myAwesomeReducer', {
+			globalSelector: ( state ) => state,
+		} );
+		const unsubscribe = subscribe( () => {
+			data = select( 'myAwesomeReducer', 'globalSelector' );
+		} );
+		const action = { type: 'dummy' };
+		store.dispatch( action ); // increment the data by => data = 2
+		store.dispatch( action ); // increment the data by => data = 3
+
```

Comment:
I would insert an expectation here too.

Reply:
Why it's the same value as the last one? The idea for the two remaining dispatches is that it doesn't increment anymore.

Reply:
I was talking about this:

```js
const action = { type: 'dummy' };

store.dispatch( action );
expect( inrecementedValue ).toBe( 2 );

store.dispatch( action );
expect( inrecementedValue ).toBe( 3 );

unsubscribeFromIncrementing();

store.dispatch( action );
store.dispatch( action );

expect( inrecementedValue ).toBe( 3 );
```

Reply:
Why not! Adding